phc/web/analytics.py

```diff
@@ -0,0 +1,17 @@
+import json
+
+from phc.web.base_client import BaseClient
+
+
+class Analytics(BaseClient):
+    """Provides analytics subject search"""
+
+    def jprint(self, data):
```

Comment:
Think this should live on the `ApiResponse`? Then we could pretty print any response.